commit 3d37564726c422c567577d2263e7f10ed3d519ae (refs/changes/90/3190/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-01-11T01:05:42+00:00 (1 year, 9 months ago)
    
    TRITON-45 VOLAPI uses wrong error level when logging successful import of storage VM image
    Reviewed by: Nick Zivkovic <nick.zivkovic@joyent.com>
    Approved by: Nick Zivkovic <nick.zivkovic@joyent.com>

diff --git a/lib/volapi.js b/lib/volapi.js
index e67abb9..f69dd1a 100644
--- a/lib/volapi.js
+++ b/lib/volapi.js
@@ -207,7 +207,7 @@ function importImage(imgapiClient, imageUuid, log, callback) {
                                 'Error when importing image');
                             importImgBackoff.backoff();
                         } else {
-                            log.error({imageUuid: imageUuid},
+                            log.info({imageUuid: imageUuid},
                                 'Imported image succesfully');
                             importImgBackoff.reset();
                             callback();
